addons/point_of_sale/tests/test_frontend.py

```diff
@@ -1563,6 +1563,17 @@ def test_product_categories_order(self):
         self.main_pos_config.with_user(self.pos_admin).open_ui()
         self.start_tour(f"/pos/ui?config_id={self.main_pos_config.id}", 'PosCategoriesOrder', login="pos_admin")
 
+    def test_pos_save_orderlines_IndexedDB(self):
+        """ Test that the orderlines are saved in the IndexedDB """
+        self.env['product.product'].create({
+            'name': 'Test Product 1',
+            'available_in_pos': True,
+            'list_price': 1,
+            'taxes_id': False,
+        })
+        self.main_pos_config.with_user(self.pos_admin).open_ui()
+        self.start_tour(f"/pos/ui?config_id={self.main_pos_config.id}", 'PosSaveOrderlinesIndexedDB', login="pos_admin", watch=True)
```

Comment:
you
```suggestion
        self.start_pos_tour('PosSaveOrderlinesIndexedDB')
```
Why do you need to use `pos_admin`?

Reply:
It wasn't necessary, but it wasn't an issue. Anyway, I modified it.